### ISSUE TITLE
Deprecated options used in configuration files

### ISSUE THREAD

**Issue by microbug** [cryptostorm/cryptostorm_client_configuration_files#24]
From Tunnelblick:

```
Warning: This VPN may not connect in the future.

The OpenVPN configuration file for 'cstorm_****' contains these OpenVPN options:

'comp-lzo' was deprecated in OpenVPN 2.4 and removed in OpenVPN 2.5
'key-method' was deprecated in OpenVPN 2.4 and removed in OpenVPN 2.5
'ns-cert-type' was deprecated in OpenVPN 2.4 and removed in OpenVPN 2.5

You should update the configuration so it can be used with modern versions of OpenVPN.

Tunnelblick will use OpenVPN 2.4.4 - OpenSSL v1.0.2n to connect this configuration.

However, you will not be able to connect to this VPN with future versions of Tunnelblick that do not include a version of OpenVPN that accepts the options.
```

**Comment by df-cryptostorm** [comment]
We're aware that these options are deprecated and will be removed soon.
I just put up the configs of our new ECC instances @ https://github.com/cryptostorm/cryptostorm_client_configuration_files/tree/master/ecc
which includes updated options and stronger crypto.

We can't completely remove the configs that contain deprecated options just yet because a lot of customers are still using devices that run OpenVPN 2.3.x, and some of those devices make it difficult to upgrade OpenVPN.
Eventually, probably when OpenVPN 2.5 is released, we will drop support for OpenVPN 2.3.x and switch the ECC instances over to the old ones.

**Comment by GradeyCullins** [comment]
Nice!
